requirements/dev.txt

```diff
@@ -150,7 +150,7 @@ requests-toolbelt==0.9.1
     # via twine
 rfc3986==2.0.0
     # via twine
-rich==12.5.1
+rich==12.6.0
```

Comment:
No breaking change in the new version: https://github.com/Textualize/rich/blob/master/CHANGELOG.md#1260---2022-10-02